### DISCUSSION TITLE
请程序高手写一个V3版的开机启动的程序

### DISCUSSION THREAD

**Discussion by zotojn269** [AlistGo/alist#2035] [Q&A]
请程序高手写一个V3版的开机启动的程序

**Comment by mixpplus** [comment]
1. 官网有windows的守护进程vbs脚本
2. 脚本写完以后可以自己写一个bat脚本用来执行vbs脚本并把bat脚本放在`C:\ProgramData\Microsoft\Windows\Start Menu\Programs\Startup`该目录下
bat脚本格式：
```bat
@echo off
C:
cd C:\soft\alistDir
start.vbs
```
3. 重新启动系统即可

**Comment by anwen-anyi** [comment]
**[https://www.bilibili.com/video/BV1kP411J7YN/](https://www.bilibili.com/video/BV1kP411J7YN/)**，拿去不谢
